ui/app/styles/core/charts.scss

```diff
@@ -346,3 +346,35 @@ p.data-details {
     grid-row-start: 4;
   }
 }
+
+// LINEAR STYLING //
+.lineal-chart {
+  position: relative;
+  padding: 10px 0px 10px 50px;
+  svg {
+    overflow: visible;
+  }
+}
+.lineal-chart-bar {
+  fill: $blue-500;
```

Comment:
Sorry for the delay on merging my PR - I [updated these](https://github.com/hashicorp/vault/pull/24852/files#diff-fc121aee03c52c0989a5113e08e2e3cac83a645edf3b97589ad07c2f1b07f539) to use HDS tokens, could you update this to ` var(--token-color-palette-blue-300);` so they match?